### PR TITLE
Fixes #5861: service_ensure_stopped_at_boot is missing

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/5861

Replacing previous PR: https://github.com/Normation/ncf/pull/146

### CODE REVIEW

**Comment by jooooooon** [comment]
Note for self: review this

File: tree/30_generic_methods/service_check_stopped_at_boot.cf [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#####################################################################################
+# Copyright 2014 Normation SAS
```
Comment (ncharles) [inline tree/30_generic_methods/service_check_stopped_at_boot.cf]: shouldn't it be 2016 ?

File: tree/30_generic_methods/service_check_stopped_at_boot.cf [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  vars:
+
+    systemctl_utility_present::
+      "command_to_check"   string => "${paths.path[systemctl]} is-enabled ${service_name}; test $? -ne 0";
+
+    !systemctl_utility_present.chkconfig_utility_present::
+      "command_to_check"   string => "${paths.path[chkconfig]} --list ${service_name} | grep -q 3:off";
+
+    !systemctl_utility_present.!chkconfig_utility_present::
+      "command_to_check"   string => "${paths.path[test]} -f /etc/rc`runlevel | ${paths.path[cut]} -d' ' -f2`.d/K??${service_name}";
```
Comment (ncharles) [inline tree/30_generic_methods/service_check_stopped_at_boot.cf]: what about chitab and svcadm ?

Comment (ncharles) [inline tree/30_generic_methods/service_check_stopped_at_boot.cf]: like in service_ensure_at_boot ?

File: tree/30_generic_methods/service_ensure_stopped_at_boot.cf [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#####################################################################################
+# Copyright 2014 Normation SAS
```
Comment (ncharles) [inline tree/30_generic_methods/service_ensure_stopped_at_boot.cf]: it should be 2016

File: tree/30_generic_methods/service_ensure_stopped_at_boot.cf [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+{
+  vars:
+
+    systemctl_utility_present::
+      "command_to_register" string => "${paths.path[systemctl]} disable ${service_name}";
+
+    !systemctl_utility_present.chkconfig_utility_present::
+      "command_to_register" string => "${paths.path[chkconfig]} ${service_name} off";
+    !systemctl_utility_present.update_rcd_utility_present::
+      "command_to_register" string => "${paths.path[update_rcd]} ${service_name} disable";
```
Comment (ncharles) [inline tree/30_generic_methods/service_ensure_stopped_at_boot.cf]: same here, missing solaris and AIX support

**Comment by amousset** [comment]
@peckpeck This PR needs to be updated according to the latest comments.

**Comment by amousset** [comment]
PR rebased

**Comment by amousset** [comment]
Done in #534

**Pull request closed without merging**